ocs_ci/ocs/ocp.py

```diff
@@ -844,3 +845,105 @@ def verify_images_upgraded(old_images, object_data):
         f"All the images: {current_images} were successfully upgraded in: "
         f"{name}!"
     )
+
+
+def check_upgrade_completed(self, target_version):
+    """
+    Check if OCP upgrade process is completed:
+        checks if all ClusterOperator images updated
+
+    Args:
+        target_version (str): expected OCP client
+
+    Returns:
+        bool: True if success, False if failed
+
+    """
+    log.debug(f"self.get_current_oc_version: {self.get_current_oc_version()}")
+    log.debug(f"target_version=: {target_version}")
```

Comment:
should be log.info

Also, change to:
f"The target version is: {target_version}"